src/shared/ui/modal-window/modal-window.vue

```diff
@@ -0,0 +1,17 @@
+<script setup lang="ts">
+import { modal, modalClose } from '@/shared/lib/modal'
```

Comment:
Внутри одного сегмента следует использовать относительные пути для импорта.

```suggestion
import { modal, modalClose } from '../../lib/modal'
```